packages/pwa-kit-react-sdk/bin/pwa-build.js

```diff
@@ -19,16 +20,17 @@ const webpack = path.join('node_modules', '.bin', 'webpack')
 sh.set('-e')
 sh.config.silent = true
 
+const config = Utils.getConfig() || {}
```

Comment:
I don't have time to review this thoroughly this minute, but these changes will affect V2. Can we sync up on the changes before merging please @bendvc ?

Reply:
Small summary of what we talked about IRL. Oli's concern was whether or not we can implement in v2 and if it's possible to maintain the same API for getting configuration in V2 as we have in this PR. This resulted in my doing a small spike where I was able to get configuration integrated quite easily, the API however is a little different based on how the new mono repo is broken into more projects, the express server definition doesn't exist along side of the react rendering and overall having the utility split up felt more natural to me. It's probably possible to get the same API using webpack plugins to include the cosmiconfig module, but that isn't ideal.